org.opentosca.bus/org.opentosca.bus.application.api.soaphttp/src/main/java/org/opentosca/bus/application/api/soaphttp/route/Route.java

```diff
@@ -16,14 +16,14 @@
  * Route of the Application Bus-SOAP/HTTP-API.<br>
  * <br>
  * <p>
- * The endpoint of the SOAP/HTTP-API is created here. Incoming requests will be un/marshalled,
- * routed to processors or the application bus in order to handle the requests.
+ * The endpoint of the SOAP/HTTP-API is created here. Incoming requests will be un/marshalled, routed to processors or
+ * the application bus in order to handle the requests.
  *
  * @author Michael Zimmermann - zimmerml@studi.informatik.uni-stuttgart.de
  */
 public class Route extends RouteBuilder {
 
-    private final static String ENDPOINT = "http://0.0.0.0:8084/appBus";
+    private final static String ENDPOINT = "http://0.0.0.0:8082/appBus";

```

Comment:
Why were the ports changed?

Reply:
that port is used in the github actions runner by something else...sadly